From dc984890a0e56ca6122dd2bd8e0b3cfe69031ced Mon Sep 17 00:00:00 2001
From: Mathieu Velten <matmaul@gmail.com>
Date: Fri, 2 Mar 2018 18:50:44 +0100
Subject: [PATCH] lua: fix loading file when the fs always reports mtime = 0

---
 share/lua/intf/http.lua | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/share/lua/intf/http.lua b/share/lua/intf/http.lua
index 1cf8946126..5367ea08c1 100644
--- a/share/lua/intf/http.lua
+++ b/share/lua/intf/http.lua
@@ -93,9 +93,9 @@ function process(filename)
     local func = false -- process_raw(filename)
     return function(...)
         local new_mtime = vlc.net.stat(filename).modification_time
-        if new_mtime ~= mtime then
+        if func == false or new_mtime ~= mtime then
             -- Re-read the file if it changed
-            if mtime == 0 then
+            if func == false then
                 vlc.msg.dbg("Loading `"..filename.."'")
             else
                 vlc.msg.dbg("Reloading `"..filename.."'")
@@ -229,9 +229,9 @@ function rawfile(h,path,url)
     local page = false -- io.open(filename):read("*a")
     local callback = function(data,request)
         local new_mtime = vlc.net.stat(filename).modification_time
-        if mtime ~= new_mtime then
+        if page == false or new_mtime ~= mtime then
             -- Re-read the file if it changed
-            if mtime == 0 then
+            if page == false then
                 vlc.msg.dbg("Loading `"..filename.."'")
             else
                 vlc.msg.dbg("Reloading `"..filename.."'")
-- 
2.14.3

